client/app/helpers/optionset.js

```diff
@@ -81,6 +81,19 @@ const OPTIONSET_LOOKUP = {
     dcpNewfacilityopt: COMMON_OPTIONSETS.YES_NO_INTEGER,
     dcpIsprojectlistedinstatementofneedsopt: COMMON_OPTIONSETS.YES_NO_INTEGER,
     dcpDidboroughpresidentproposealternativesite: COMMON_OPTIONSETS.YES_NO_INTEGER,
+    dcpRelatedacquisitionofpropertyopt: LANDUSE_FORM_OPTIONSETS.DCPRELATEDACQUISITIONOFPROPERTYOPT,
```

Comment:
can this just be assigned `COMMON_OPTIONSETS.YES_NO_INTEGER`?

Reply:
yep!